src/BridgedRoom.ts

```diff
@@ -47,6 +47,9 @@ interface ISlackChatMessagePayload extends ISlackToMatrixResult {
     icon_url?: string;
 }
 
+const RECENT_MESSAGE_MAX = 10;
+const PUPPET_INCOMING_DELAY_MS = 1500;
```

Comment:
This seems high? I suppose it's only for messages a logged in user sends from slack?

Reply:
yep.